control-plane/rest/openapi-specs/v0_api_spec.yaml

```
@@ -1604,6 +1604,50 @@ paths:
           $ref: '#/components/responses/ServerError'
       security:
         - JWT: [ ]
```

Seems convention in existing code is `[]` rather than `[ ]`
```suggestion
        - JWT: []
```

---

Ohh right.